.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+
+[*.php]
+indent_style = tab
+indent_size = 8
```

Comment:
I thought that indent size is 8 when I look at GitHub, but we can change this number according to your preference.